### DISCUSSION TITLE
Mania Argon: Slider ends are very hard to read

### DISCUSSION THREAD

**Discussion by yuuiko** [ppy/osu#21996] [Ideas] [answered]
Latest iteration of argon mania looks good but feels super cluttered.

Slider ends especially need some cleanup.

<img width="389" alt="Screenshot 2023-01-02 at 12 35 18 pm" src="https://user-images.githubusercontent.com/50823728/210189902-eaefb2ca-b34c-488c-8d0e-f6ca589a6136.png">

It's very hard if not impossible to discern what is a normal note, a hold note, and how long sliders are when the pattern is dense

I suggest removing the darker shade on top of all notes, and greatly simplifying the slider end, perhaps no slider end is needed and it'll just be the background of the slider.

Here's a quick mockup:
<img width="389" alt="Screenshot 2023-01-02 at 12 35 18 pm" src="https://user-images.githubusercontent.com/50823728/210190185-82be490c-3d94-46c2-9477-88bb64a792ce.png">

**Comment by peppy** [comment]
Thanks for the feedback. Your design looks quite good, so I'd definitely be interested in seeing a PR with this implemented.

I do wonder if we still want some kind of bar at the actual hit location (ie. where the solid white box is) though.

**Reply by yuuiko** [reply]
As a former skinner I would love to help out though I'm not familiar with dev of any kind. Would there be a few resources that would show me how to get started with contributing?

**Reply by peppy** [reply]
Your best bet would be to provide mock-up images.

Do note that a change has already been pushed out and will be in the next release. So I'd wait to see that before making any further changes.

**Reply by yuuiko** [reply]
I have responded! https://github.com/ppy/osu/discussions/21996#discussioncomment-4929241

**Comment by itsMapleLeaf** [comment]
this post is old, see new proposal at https://github.com/ppy/osu/discussions/21996#discussioncomment-4753711

<details>
<summary>old content</summary>
Here's my proposal: a simple line for the hold end. I've play-tested this a bit. It's nicer to read than current Argon, and the tail is clear enough for lower skill levels.

![dotnet_ecfAiiG639](https://user-images.githubusercontent.com/19603573/213896489-68489ba7-3adb-46fa-b3ee-515b50f1e507.png)

Here's an alternate variant, with the background extended to match the height of notes. This one is a bit visually noisier and harder to play with

![dotnet_ZUv964ioQt](https://user-images.githubusercontent.com/19603573/213896485-a32f1f60-2d16-4555-980f-240053b9fa16.png)

...and it suffers from an issue where it can overlap with other upcoming hit objects:
![image](https://user-images.githubusercontent.com/19603573/213896606-54573516-1e06-495b-80e8-8ad39702cc1a.png)
</details>

**Reply by peppy** [reply]
The problem with the single line is it may become harder to understand when to release (since you're releasing based on only a line rather than the whole block aligning with the judgement area.

**Reply by itsMapleLeaf** [reply]
Sure, I thought about that. But in practice, having a single line means every hold intuitively occupies the vertical length respective of how long to hold it. In other words, a 1/1 hold actually looks like a 1/1 hold, instead of a 1/1 hold + some change

In contrast, note-sized hold ends make holds look much longer than they actually are

I also realize that seeing things this way might be a matter of perspective. I've played mania for a long time, so I read hold lengths, not hold releases, and, looking at the landscape of pro skins, I'm probably not alone. What if we implemented this single-line version just in Argon pro?

EDIT: I've been thinking and asking around, and I think I'm a little short-sighted in some of these points. In isolation, knowing exactly when to release a note without another reference to it is valuable, and the single non-extended line does not do that.

I'm also still not sold on the gradient though 😅 but I have some ideas

**Reply by itsMapleLeaf** [reply]
New idea: https://user-images.githubusercontent.com/19603573/213945610-5c8d9743-3812-4cef-9ae5-63be444e6d57.png

Something less noisy that won't be easily confused for notes and hold heads, and I tried to still make it clear when you need to release

Here are some alternatives:
https://user-images.githubusercontent.com/19603573/213947514-84d7cf85-2c50-462a-9fd2-88fb68851c77.png
https://user-images.githubusercontent.com/19603573/213947517-0ddc45eb-2a30-415e-ba7c-ba5387916f3f.png

**Reply by peppy** [reply]
The line is in the wrong place on the first two.

**Reply by itsMapleLeaf** [reply]
The line position on those was a stylistic choice, but I get what you mean. I like the third one the most too at the moment. Would you accept a PR for that?

EDIT: Here's the third style with fixed visual artifacts 
https://user-images.githubusercontent.com/19603573/213975343-da039b31-a6a6-45aa-8a26-d88395742518.png

**Reply by peppy** [reply]
I'm not against it, but I'm also not sure it is going to solve the issue have with too much visual noise.

And I still quite like the gradient approach in the opening post, potentially with an added hint as to where the judgement line is.

**Reply by itsMapleLeaf** [reply]
> I'm not against it, but I'm also not sure it is going to solve the issue have with too much visual noise.

I thought the same! But the darkened tail actually does help a lot in practice. On high scroll speeds and dense LN maps, they sort of visually fade out, and you focus more on the hold bodies

Here's what it looks like in action:
https://user-images.githubusercontent.com/19603573/213979037-9d3520d5-6c98-4bb2-9d38-6d4581a6d68e.mp4

**Reply by peppy** [reply]
I'm not a huge fan. I don't like the icon being there (it should either not be there at all or, be something different like a dot or a short horizontal line to differentiate from the start of the hold.

**Comment by itsMapleLeaf** [comment]
Dug into Figma to create a bunch of different variations I've thought of. The single hit line and shade variants I think have the best balance of aesthetics and clarity.

![proposals for mania holds in argon](https://user-images.githubusercontent.com/19603573/215380570-d322a81e-276a-4c77-83d8-d769b2531f42.png)

---
<details>
<summary>old content</summary>

Experimented a bit, here's what the gradient looks like with a hit line added:
<img src="https://user-images.githubusercontent.com/19603573/213984198-3e85db35-829d-4110-a0af-f50cfcad9b55.png" alt="" height=500 />

I think a white hit line would still stand out and look too much like a note, so I went with an accent-colored hit line

I personally still prefer the darkened end:
<img src="https://user-images.githubusercontent.com/19603573/213975343-da039b31-a6a6-45aa-8a26-d88395742518.png" alt="" height=500 />

Here, without the arrow on the tail:
<img src="https://user-images.githubusercontent.com/19603573/213988054-78636b82-90c5-4de6-8340-95fa598839a5.png" alt="" height=500 />

</details>

**Reply by itsMapleLeaf** [reply]
@peppy Moving to this new reply because the old one got cluttered and derailed. I'll keep this one up-to-date with the latest status of the convo

Thoughts? Do you still prefer the gradient?

(sorry in advance if I'm being too "all over the place" with this!)

**Comment by arflyte** [comment]
After looking at the designs above, I liked the simple no-shade the best so far; it'll make everything less busy.
![chrome_230130-160826](https://user-images.githubusercontent.com/3632750/215410418-b6cbae18-6cd3-4bc3-b3ed-cabdb3f28d90.png)

I thought the gradient was nice too at first, but that too might be too busy for some pro maps.

**Reply by peppy** [reply]
Let's give it a try with no markings then, and see what kind of feedback we get.

**Reply by itsMapleLeaf** [reply]
updated my PR! https://github.com/ppy/osu/pull/22402

**Reply by itsMapleLeaf** [reply]
one minor qualm after trying this out in gameplay: LNs look like they last longer than they actually do, and take up a lot of vertical space

compare this side-by-side with the zero-length version, where each LN visually looks as long as it lasts (each a 1/4 beat in this case)

I know this is one tradeoff for another though. This is why I liked the darkened tail, because it makes the tail stand out less, helping readability on denser maps, but they're still visible for when you want clarity on when to release

| flat note-sized tail | flat zero-length tail | dark tail | dark tail,<br>bottom-aligned line |
| --- | --- | --- | --- |
| <img src="https://user-images.githubusercontent.com/19603573/215553232-a5c36acf-4553-4ec3-8dff-dfc4a07102c8.png" width="200" /> | <img src="https://user-images.githubusercontent.com/19603573/215553161-898fa4cf-98f0-48fa-90e4-cbfabd6af984.png" width="200" /> | <img src="https://user-images.githubusercontent.com/19603573/215557074-9663d8a6-a7db-412d-aca6-67ad1144dc40.png" width="200" /> | <img src="https://user-images.githubusercontent.com/19603573/215563152-421b694d-a03e-4a54-9e56-9e7d03a98244.png" width="200" /> |

**Reply by peppy** [reply]
I still personally prefer 3 or 4.

**Reply by itsMapleLeaf** [reply]
Alright, I'll go with 3 because it fits the judgement area and makes the most "visual sense" while doing so (a shadow-extended tail with a bottom line looks weird lol)

**Reply by peppy** [reply]
Sure, please update the PR to match.

**Reply by itsMapleLeaf** [reply]
updated!

**Comment by itsMapleLeaf** [comment]
Also... the "shadow" on the tap notes (the dark line above them) may or may not be throwing some wrenches into this equation. It raises a whole separate question of whether that's actually a part of the full note height or not, and I've been unsure of whether to pull the hold body to meet that, or just meet the bright part of the note

![where should the body end?](https://user-images.githubusercontent.com/19603573/215585883-4913db1e-e5e6-426d-ada9-1d2453da32d3.png)

**Reply by peppy** [reply]
The "shadow" fits within th judgment area..

**Reply by yuuiko** [reply]
Can we just remove that part or at least make it an actual shadow with a black→transparent gradient, because i keep thinking that part is a really short slider

**Comment by yuuiko** [comment]
## OK new update is out, here is my response.

<img width="1230" alt="Fixing Argon Mania" src="https://user-images.githubusercontent.com/50823728/218038463-b450f46c-ef21-4551-b133-f866be59970c.png">

⚠️ Specs have been updated, please review comments.

Colours have changed, now it's:
<img width="1198" alt="Screenshot 2023-04-04 at 3 06 53 pm" src="https://user-images.githubusercontent.com/50823728/229692311-b20c70d4-04cf-439b-b438-bb4ed49a5f44.png">

**Reply by peppy** [reply]
cc @arflyte

**Reply by itsMapleLeaf** [reply]
I felt fairly satisfied with the update I pushed through, but this looks really nice too!

**Reply by arflyte** [reply]
From design perspective, I like at what I'm looking here.

**Reply by peppy** [reply]
I'm on board with all of these changes, with one exception: can we make the shadow the same as the old one but semi-transparent, rather than the drop shadow style you've change it to? I think with your change, it's giving too much depth to the notes, which doesn't match the look of argon.

As far as I can tell, this would not detract from readability too much.

**Reply by itsMapleLeaf** [reply]
What if we went without the gradients and fades entirely? Give holds the outline without the faded tip, keep the old single-color shadow on notes

![](https://user-images.githubusercontent.com/19603573/218649529-4cdf4734-fc28-4b91-9e0d-531a221f52b8.png)


Basically just ironed what's in the proposal, lol

**Reply by peppy** [reply]
I prefer what's the proposal. Also the inner edges of your stroke not having corner radius feels very wrong.

**Reply by itsMapleLeaf** [reply]
Take two. There are some color alterations in the proposal near the tail that I couldn't figure out 😅 but this is just a general idea

![](https://user-images.githubusercontent.com/19603573/218656835-a01da183-9f9a-41d9-9aa6-9754604daa24.png)

**Reply by peppy** [reply]
I don't understand what your'e pushing for. Let's go with the proposed one. Unless you have a *reason* to push for a change (please state the reason).

**Reply by itsMapleLeaf** [reply]
I was visualizing what you said here for clarity:

> can we make the shadow the same as the old one but semi-transparent, rather than the drop shadow style you've change it to?

**Reply by peppy** [reply]
The shadows are the pieces underneath the note (at the end), which aren't in your examples:

![Safari 2023-02-15 at 03 07 01](https://user-images.githubusercontent.com/191335/218916946-fe822336-1ad8-4bbf-a4a4-27a1bc9de33d.png)

![Safari 2023-02-15 at 03 07 33](https://user-images.githubusercontent.com/191335/218916996-2f96ff26-8c2f-4840-b733-f81e75504861.png)

**Reply by itsMapleLeaf** [reply]
What? No, they _are_ there, it's  the shading inside of the hold body

![image](https://user-images.githubusercontent.com/19603573/218942501-57bcb8f5-bce9-4648-964a-1a1c6c89fbc1.png)

But I guess this confusion is another way of saying it should overlap the hold body outline instead of being inside of it. I thought that felt weird and not very cohesive, but 🤷

**Reply by yuuiko** [reply]
> I'm on board with all of these changes, with one exception: can we make the shadow the same as the old one but semi-transparent, rather than the drop shadow style you've change it to? I think with your change, it's giving too much depth to the notes, which doesn't match the look of argon.
> 
> As far as I can tell, this would not detract from readability too much.

Quick mockup with flatter shadows

<img width="860" alt="Screenshot 2023-02-16 at 8 31 50 pm" src="https://user-images.githubusercontent.com/50823728/219325775-17bb70e1-19d8-42bf-be54-28739a2f7c8d.png">

It does work better than current argon though the hard lines make it a bit less visually intuitive

<img width="921" alt="Screenshot 2023-02-16 at 8 31 41 pm" src="https://user-images.githubusercontent.com/50823728/219326124-02c67d21-a89b-4863-b245-dd39fac93a28.png">

Shortening the shadow seems to make it better.


In extreme cases, the hard line makes it very hard to read, but the notes would need to be VERY close together and thats quite rare unless its a trickmap, so statistically this compromise can be made without much affect (these are the shorter shadows)
<img width="1039" alt="Screenshot 2023-02-16 at 8 31 08 pm" src="https://user-images.githubusercontent.com/50823728/219326394-4cdcbc4f-a010-4eda-9439-9f70ae60f23a.png">



One major point is to not add any additional harsh lines to the top of the slider head, because this is what the slider end will match up to. (though I suppose the transparent shadow makes it not too bad though)

**Remember the idea is that the shadow should only be noticeable when notes overlap, while fading away when notes don't as to reduce visual clutter.**

In the image below, one might intuitively but incorrectly release when B hits C instead of when B hits D.

<img width="830" alt="Screenshot 2023-02-16 at 8 42 49 pm" src="https://user-images.githubusercontent.com/50823728/219328630-7be014eb-3a36-4d12-a077-f321777947aa.png">




Also consider https://github.com/ppy/osu/discussions/21994#discussioncomment-4994425

**Reply by peppy** [reply]
~~@yuuiko what are your thoughts on changing lane colours to be mirrored from centre outwards (on non-special/scratch layouts)? this seems more standard and has been brought up in discussion.~~

Never mind, this is already covered in your graphics.

**Reply by peppy** [reply]
@yuuiko could you explain the choice of the lighter purple (lavender?) shade in 3k? It's one of the choices that feels a bit out of place as

- In all other cases it's uses for the special key
- It's next to the other purple

![Safari 2023-03-08 at 10 52 49](https://user-images.githubusercontent.com/191335/223694380-7754175e-9c08-419e-a504-081ba2caf075.png)

**Reply by yuuiko** [reply]
> could you explain the choice of the lighter purple (lavender?) shade in 3k? It's one of the choices that feels a bit out of place as

Agreed, I like what you did in the update instead.
Sorry i didn't see your messages earlier, didn't seem to get pinged for anything on my end, my bad.

The new mania update is so much better and working great! though I realised I botched some colour choices after actually play-testing.

Here are improved colours that take consistency into mind and better difference of colours. Each colour should now generally match a finger across keycounts with some popular exceptions of which are more optimised for that keycount
<img width="1159" alt="Screenshot 2023-03-27 at 11 28 24 am" src="https://user-images.githubusercontent.com/50823728/227814817-0d9c0861-f434-4418-99e1-2852ae1a7a9c.png">


In addition I found the yellow to be still a bit too bright, so I've refined it further.
<img width="268" alt="Screenshot 2023-03-27 at 11 12 15 am" src="https://user-images.githubusercontent.com/50823728/227814349-11cb2d1a-3754-4495-aa91-528d702115da.png">

I would appreciate any pointers on how to get started with lazer development, so I could contrib more than mockups! I have done a few programming projects, and I have a high standard of quality and best practice

**Reply by Alsen99** [reply]
@yuuiko why is 6k fifth lane is purple and not orange like second lane?

**Reply by yuuiko** [reply]
> @yuuiko why is 6k fifth lane is purple and not orange like second lane?

a bit like 5K accenting left and right hands, idk
I like the current 6K in lazer right now but it was just the middle two that was off

suppose it could be orange too for consistency but for my personal pref i like it spiced up a tiny bit

**Reply by TranT3011** [reply]
my opinion is like half subjective half objective?

i learned mania with these colors https://cdn.discordapp.com/attachments/1089967641550143548/1089967780830392340/screenshot1920.jpg
i think those look ok 
so when im looking at the new colors, im kinda not sure why they are different from how mania players are used to and is changing them a good idea for new players?
i mean it isn't colors but lane pairs having the same colors
i honestly dont see a reason to change that pairs when 100% mania get used to it

probably need more mania players' opinion

**Reply by TranT3011** [reply]
me and another top mania player agree that the skin is trying to be very colorful which fits lazer and not really playable (subjective)
i think being colorful is nice but it should have some playability
https://cdn.discordapp.com/attachments/1089967641550143548/1089979550949195817/image.png

**Reply by yuuiko** [reply]
@TranT3011 I believe its possible to get it to a state where its reasonably readable enough while still fitting lazer's theme. For example, idk but IMO I think 5K is just right

Just get good at design, and to do that, we need feedback

Interestingly I did do the pattern u showed originally, but perhaps the current colours contrast too much which is what makes it distracting.

for 7K, how about something like
red orange red purple red orange red

another thing to say is, of course everyone is used to their own skin. so we need to be self aware about familiarity bias, but at the same time, it should be a certain amount reasonably intuitive

hence my colour saturation reduction and everything in my design doc

**Reply by Alsen99** [reply]
I believe the solution already exists, the separation of Argon and Argon Pro skin.

The normal Argon can be colorful, and the Argon Pro are meant for pro players. Or else separating them would be useless.

I also want to play using Argon Pro and I agree that too much color might be too distracting and should just stick with color schemes that pro players would use

Also the width of low key counts can be wider for pro too

**Reply by yuuiko** [reply]
@Alsen99 Agreed, though I want the standard skin to be as readable as possible while still having the "fun" feel. Argon Pro is where we can give up some of the "fun" feel for hardcore optimisation, no icons, bright notes, seperate coloured hold notes etc.

I mean, I can't speak for what peppy wants, but this is what I think.

An updated Colour spec, what do you think? after a few playtests and feedback.
This time I reduced some overly contrasting combinations e.g. 6K, 7K and observed tradition.
I do admit that I'm not a 7K player but I can see how 7K would be played.

7K has blue middle because it seems most people want that accented, purple is a warm colour and wouldn't contrast as much. Blue in this design is also perceivably a little brighter than purple

<img width="1054" alt="Screenshot 2023-03-28 at 1 14 24 pm" src="https://user-images.githubusercontent.com/50823728/228109886-fb8b2357-a175-4e97-84dd-4c9f173ad862.png">

**Reply by TranT3011** [reply]
i think we can balance the readability and color at the same time tho
cuz like what if a new player keeps playing the not very good skin he will struggle to play it

**Reply by peppy** [reply]
please keep "argon pro" discussions out of this thread. it has a very meaningful existence, but your discussion is at this point 100% off topic. focus on the proposal in the thread and nothing further.

**Reply by Alsen99** [reply]
> please keep "argon pro" discussions out of this thread. it has a very meaningful existence, but your discussion is at this point 100% off topic. focus on the proposal in the thread and nothing further.

My bad, I'll delete my comment

**Reply by Alsen99** [reply]
@yuuiko Shouldn't the 9k middle note be blue too?

**Reply by peppy** [reply]
The only issue I have is that I would still prefer that the "large" / special / space bar note is a consistent colour (ie. always lilac).

**Reply by Alsen99** [reply]
I just noticed that the 8k is wrong, after looking at most popular 8k beatmaps, they all use IIDX style which means its more like 7k + 1, which means that the most left or right is actually a special/bigger note like 7k middle note.

I guess it was never intended to be played that way, in that case the 8k is correct

**Reply by yuuiko** [reply]
> The only issue I have is that I would still prefer that the "large" / special / space bar note is a consistent colour (ie. always lilac).

@peppy what do u think about blue middles and axe lilac?

blue/green was kinda used as "the middle ones" in a lot of them (but 5K and 6K is thought of differently so disregard etc.)

Basically what @Alsen99 said about 9K

**Reply by peppy** [reply]
> I just noticed that the 8k is wrong, after looking at most popular 8k beatmaps, they all use IIDX style which means its more like 7k + 1, which means that the most left or right is actually a special/bigger note like 7k middle note.
> 
> I guess it was never intended to be played that way, in that case the 8k is correct

We still have intention of supporting the 7+1 layout in lazer, in which case the special note would be moved to the left (but the colour unchanged), so please factor that in to colour choices.

> @peppy what do u think about blue middles and axe lilac?

I was kinda fond of the lilac colour since it's one of those "unique" colours that makes it feel suited to a special key. More than blue anyway. But that's only a weak preference.

**Reply by Alsen99** [reply]
Here is my proposal, just a disclaimer that I'm not actually a pro player in a sense of high ranking, but I do play 4k mania for a long time with a lot less hours in 6k and 7k. So, if my suggestion is not used, I'm ok with it, just want to progress this discussion by pitching ideas

1k: red
Explanation: any color is fine in my opinion, its 1 button after all.

2k: orange, red

3k:
First option: orange, lilac, orange
Explanation: I am assuming the middle one uses thumb

Second option: orange, red, orange
Explanation: This is more general since 3k can be played index, index, middle or index, middle, ring (one hand)

4k:
First option: orange, red, red, orange
Second option: orange, orange, red, red
Explanation: having color pattern actually helps reading in my experience so I prefer either of this otherwise yuuiko's design is ok

5k: 
First option: orange, red, lilac, red, orange
Explanation: I am assuming the middle note uses thumb which make the middle one a special note

Second option: orange, red, orange, red, orange
Explanation: This is assuming the playstyle is middle, index, index, middle, ring. Just like playing Kalpa

6k: orange, red, orange, orange, red, orange

7k: orange, red, orange, lilac, orange, red, orange

7k + 1 (left is the big note): 
First option:Lilac, orange, red, orange, lilac, orange, red, orange
Explanation: this is IIDX style so the left most and middle is a special/middle key

Second option: blue, orange, red, orange, lilac, orange, red, orange
Explanation: I'm making blue the second special color, but its only used in 7k + 1 and nowhere else

8k: magenta, red, orange, yellow, yellow, orange, red, magenta
Explanation: the reason i don't use green and blue to distinguish the middle notes is that in my opinion the stage should separate the middle (like giving a visible line to indicate the middle). Otherwise yuuiko's design is ok

9k: magenta, red, orange, yellow, lilac, yellow, orange, red, magenta

10k: magenta, red, orange, yellow, lilac, lilac, yellow, orange, red, magenta
Explanation: same like 8k, separate the stage otherwise yuuiko's design is ok

Sorry I didn't make a mock up since it's yuuiko's design in the first place and I'm just swapping colors.

**Reply by yuuiko** [reply]
quick sidenote i like green and blue 1K 2K because it matches the beatmap difficulty colour gradient nicely

**Reply by yuuiko** [reply]
yet another revision after a bit more 7K playing. I think this is the most intuitive and consistent yet.

- 3K new colour scheme (purely aesthetic, idea is to match beatmap difficulty gradient, also it matches purple middle too)
- 6K focus on middle fingers: 🟩🟦🟩 🟥🟧🟥 (different left and right to differentiate the middle two columns)
- 7K focus on middle fingers:  🟥🟧🟥 🟪 🟥🟧🟥   The outer two (🟥ring 🟧middle) are also consistent with 8K above.
- 8K up more consistent - 🟪pinky, 🟥ring, 🟧middle, 🟨/🟩🟦pointer always.

<img width="892" alt="Screenshot 2023-04-01 at 2 34 27 pm" src="https://user-images.githubusercontent.com/50823728/229263798-3fc200af-4e6a-4e64-af9a-9a7c10107182.png">

**Reply by yuuiko** [reply, marked as answer]
Tried 6K a bit more, and i changed 6K to be more consistent too with the ring and middle fingers
<img width="1198" alt="Screenshot 2023-04-04 at 3 06 53 pm" src="https://user-images.githubusercontent.com/50823728/229692176-58358cf1-6ee2-47b8-b16e-252ccdc70bd7.png">

**Reply by peppy** [reply]
I've updated to the latest version in https://github.com/ppy/osu/pull/23769.

**Reply by Alsen99** [reply]
> Tried 6K a bit more, and i changed 6K to be more consistent too with the ring and middle fingers
> <img width="1198" alt="Screenshot 2023-04-04 at 3 06 53 pm" src="https://user-images.githubusercontent.com/50823728/229692176-58358cf1-6ee2-47b8-b16e-252ccdc70bd7.png">
> 

Can you explain why the 6k is 🟥🟧🟩🟦🟧🟥?
 I think 🟥🟧🟥🟥🟧🟥 would be easier to read.

**Reply by peppy** [reply]
![2023-07-17 04 06 02@2x](https://github.com/ppy/osu/assets/191335/a8600445-c06f-4ed6-9ac7-8ee3b6dd9da3)
https://osu.ppy.sh/comments/2779865

**Reply by Absolute-Zeero** [reply]
>![image](https://github.com/ppy/osu/assets/30351881/3a3222e3-313d-4297-855e-9a0bda0c11a9)
https://osu.ppy.sh/comments/2779865

Seconding the 7k issue. The side lanes feel dizzying to read with how little contrast there is. Doesn't help already needing to rely so heavily on peripheral vision with such a wide playfield. Also worth mentioning: typical 7k skins tend to give lanes 2 and 6 colors of higher saturation rather than less.
Warm colors are not my forte, but I suggest 🟨🟥🟨🟩🟨🟥🟨 to start with.

**Reply by peppy** [reply]
@yuuiko I don't suppose you're still around to give some feedback/insight into this?

**Comment by Bruno5430** [comment]
My main problem with the argon skin for mania (and for other people i talked to in discord) is not the coloring of the notes, but rather the blur it generates when scrolling and fades with the color of the playfield (also the white highlight of the notes are also hard to read with the blur)
(Blur recreated with ffmpeg because my camera registers more blur than what is percepted)
![mania argon blur notes](https://user-images.githubusercontent.com/111712756/228579875-833f158d-1d73-4827-ad2f-285c198f0020.png)

The most requested solution was to make the playfield black, so the contrast between the hold note and the playfield will increase readability, both for slow and fast scroll speds
![mania yugen blur notes](https://user-images.githubusercontent.com/111712756/228582961-0660c8d3-a935-4495-8d4e-ececd677771e.png)
Skin: [Yugen](https://osu.ppy.sh/community/forums/topics/365036)
Beatmap: [Chocolate Planet](https://osu.ppy.sh/beatmapsets/734969#mania/1579836)

My proposal for the "highlight" would be to change it from white to the color of the note, and make the body of the note the same color but darker 
This changes are not too far from what is currently in taiko
![taiko argon](https://user-images.githubusercontent.com/111712756/228585520-c5aa50c0-7c8e-4d27-b9ad-d0ffac21357e.png)

If this request are "less fun" or "less colorful", a better way to colorize the playfield would be to empathize the color of the receptor's lighting
| | |
| --- | --- |
| ![mania yugen](https://user-images.githubusercontent.com/111712756/228589509-894c42fd-3126-47f1-8fbc-c40171f9b2dd.png) | ![mania ralsei](https://user-images.githubusercontent.com/111712756/228589499-d86fb4a5-53e0-49d2-a19b-8a55c7ed4842.png) |
